.buildkite/bindists-pipeline

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
This could have a comment stating it calls the `fetch-bazel-bindist` script, configures bazelrc, and adds fetched `bazel` to $PATH.

Reply:
I've added a comment to that effect.